Adjust send input size in chat input

untreu2 opened the pull request:
<!--
  Thanks for contributing!

  Provide a description of your changes below and a general summary in the title

  Please look at the following checklist to ensure that your PR can be accepted quickly:
-->

## Description
<img width="499" height="895" alt="image" src="https://github.com/user-attachments/assets/e723bdb5-cfac-425f-a642-9d8f2bec438f" />


<!--- Describe your changes in detail -->

## Type of Change

<!--- Put an `x` in all the boxes that apply: -->

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore
- [ ] 🧪 Tests

## Checklist

<!-- Please make sure you've done the following before submitting your PR: -->

- [x] Run `just precommit` to ensure that formatting and linting are correct
- [ ] Run `just check-flutter-coverage` to ensure that flutter coverage rules are passing
- [ ] Updated the `CHANGELOG.md` file with your changes (if they affect the user experience)

Fixes #586 

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **Style**
  * Chat composer now measures single-line input height to produce a more compact, consistent appearance across devices.
  * Send button padding and size adapt to the measured input height, aligning touch area with the field for improved visual balance and interaction.
  * Initial layout measurement runs after first render to ensure stable sizing.
* **New Features**
  * Compact input size option supported for a smaller field appearance.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

josefinalliende reviewed (approved):
✅  LGTM!  The arrow size looks right now! The button I still feel it is a bit smaller in height than the input, but the increased size of the arrow makes it look like less difference, so its ok for my sight! 👌🏻

Quwaysim commented:
@untreu2 I don't want to keep this PR open but I could still notice that the button is not very well up to the input field on my P8P. I'm thinking that if we merge this now, it's still going to come back to bite us 😄 Could you please have another check at it?

untreu2 commented:
Sorry, I got confused, but I got it now. Actually, I only opened this PR for the input field, but now I've fixed the send button too. I adjusted the send button size to dynamically calculate based on the single-line input to adapt to different font sizes.

Pls test, @Quwaysim.

https://github.com/user-attachments/assets/1b1d56b9-a8f4-4a0a-8d88-bd9b871010c5

josefinalliende reviewed (approved):
👍🏻  I tried it locally and now they look with the same height!

<img width="250" src="https://github.com/user-attachments/assets/e709fc16-4847-4eed-8f1d-f09d971558e8" />

erskingardner commented:
Merged since @Quwaysim is not going to be able to override his review for a while.
